src/main/scala/org/broadinstitute/dsde/workbench/sam/service/UserService.scala

```diff
@@ -58,7 +58,12 @@ class UserService(val directoryDAO: DirectoryDAO, val cloudExtensions: CloudExte
             subjectFromEmail <- directoryDAO.loadSubjectFromEmail(user.email)
             updated <- subjectFromEmail match {
               case Some(uid: WorkbenchUserId) =>
-                directoryDAO.setGoogleSubjectId(uid, user.googleSubjectId).map(_ => WorkbenchUser(uid, Some(user.googleSubjectId), user.email))
+                for {
+                  groups <- directoryDAO.listUserDirectMemberships(uid)
+                  _ <- directoryDAO.setGoogleSubjectId(uid, user.googleSubjectId).map(_ => WorkbenchUser(uid, Some(user.googleSubjectId), user.email))
+                  _ <- IO.fromFuture(IO(cloudExtensions.onGroupUpdate(groups)))
```

Comment:
shall we update google before update ldap?

Reply:
no because the google update is driven by whatever is in ldap